e2e_playwright/.gitignore

```diff
@@ -1 +1,2 @@
 performance-results/**
+test-traces/**
```

Comment:
nit: we already have the `./e2e_playwright/test-results` folder that's used for playwright test results, maybe just putting it into `./e2e_playwright/test-results/traces` is slightly cleaner.